tests/snapshot/test_dsm.py

```diff
@@ -223,7 +223,7 @@ def far_block() -> int:
 
 @pytest.fixture(scope="module")
 def far_ts() -> int:
-    return chain.time() + 14 * 24 * 60 * 60  # 14 days
+    return chain.time() + 30 * 24 * 60 * 60  # 30 days
```

Comment:
Such a time skip may cause test failures after the vote is launched due to the time-bomb item in the DG proposal.